euphonic/design/05_powder_averaging_discussion.md

```diff
@@ -0,0 +1,318 @@
+
+- Statistical methods, propogation of error approaches could provide
+  some useful tools
+- Experimentally we can examine uncertainty by comparing different
+  detector contributions to the same bin
+- We should be able to somehow examine the changing variance in S as more
+  **q**-points are sampled.
+
+<a id="org72cee07"></a>
+
+## Existing approaches
+
+
+<a id="orgce0cfe3"></a>
+
+### Random sampling
+
+-   This has been mentioned in PACE meetings. How much has it been
+    used?
+    - Random sampling on spherical shells seems more common that fully
+      random sampling in **q**-space.
+-   Has natural advantage of being uniform in space... eventually.
+-   Convergence can be examined systematically, adding new points
+    without removing old ones. A smoother process than replacing
+    entire sampling meshes with different criteria.
+-   Strictly we can sample (**q**,ω) randomly, *or* sample **q** randomly and search ω systematically at
+    each **q**.
+    - The former approach is applicable to codes like MDANSE that
+      sample S(**q**,ω) in a response formalism from dynamic data.
+    - The latter approach maps better to PACE; at each **q** we go to
+      the force constants and get all the active ω. This has the
+      advantage that every sample should find some signal, without the
+      risk of hitting "empty space".
+
+
+<a id="orgd1b667c"></a>
+
+### Spherical sampling
+
+-   SpinW seems to use a Fibonacci-related method for uniform
+    sampling at spherical surfaces
+    -   points per shell has to be a Fibonacci number
+        - That's not a deal-breaker but could be annoying when exploring convergence
+    -   or can sample *randomly* at *each* spherical shell surface
+-   MDANSE requires the user to set up a q lattice for sampling,
+    including a "spherical grid" option with a set range of "shells"
+    and "vectors".
+-   Another approach that doesn't seem to be in use but is perhaps
+    a truer estimate of the integral is to sample _within a q bin_
+    rather than exactly at the surface.
+
+
+<a id="org93896a1"></a>
+
+### Unknown
+
+-   In OCLIMAX, TASK=1 allegedly performs a "full powder
+    calculation" with coherent effects (as opposed to analytic
+    inhoherent method). This is not affected by the user q-bin
+    options, and neither manual nor paper gives any information
+    about the sampling approach.
+
+
+<a id="orgf40d9b1"></a>
+
+# Further questions
+
+-   There are two different possible objectives here: uniform sampling
+    **at each q** and uniform sampling **over the spherical
+    volume**. Which is more important?
+
+-   Why do we pre-determine the mod(q) bins?
+    -   easier to compare with instrument
+    -   able to check convergence *at each bin* independently
+    -   in Abins, the simulated *q* bins do not match the TOSCA
+        instrument measurements, in which the bin size increases with
+        *q*.
```

Comment:
If we were to randomly sample within each *q* bin, we could have arbitrary *q* bin widths which would enable us to better match the instrumental data. This would probably be much better at larger bin sizes, rather than assuming a single *q* (or a few *q*) accurately represent the instrumental measurements in that bin.

Reply:
Right, this is a challenge with binning existing randomly-distributed data: some bins might be empty or have results heavily biased by an outlier. Using large bins reduces this problem but is not guaranteed to eliminate it.